lib/ffmpeg/transcoder.rb

```diff
@@ -170,8 +172,12 @@ def delete_files(destination)
 
     def transcode_movie
       pre_encode_if_necessary
+      temp_dir = "#{TEMP_DIR}/output/"
+      FileUtils.mkdir_p(temp_dir) unless File.directory?(temp_dir)
```

Comment:
NIT since it follows so closely to the constant
```suggestion
      temp_output_dir = "#{TEMP_DIR}/output/"
      FileUtils.mkdir_p(temp_output_dir) unless File.directory?(temp_output_dir)
```